### PR TITLE
Toggle feature

### PR DESCRIPTION
**Pull request by tafodinho**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,17 +1,16 @@
...
-    const city = document.getElementById("city-text").value
-    if(e.keyCode === 13) {
-        getWeatherData(city).then((data) => {
-            renderWeatherDetails(data)
-            console.log(data)
-        })
-    }
-})
\ No newline at end of file
+inputForm.addEventListener('keypress', (e) => {
```
[the remark this answers is not in the document]
Comment (tafodinho) [reply src/index.js]: I think it was just a matter of trying to create make something different

File: src/weather.js [hunk 2]
```diff
@@ -1,19 +1,14 @@
...
-}
+const getWeatherData = (city) => fetch(`https://api.openweathermap.org/data/2.5/weather?q=${city}&units=metric&appid=e121f235d5e789b4763db2bdb5c1530e`)
+  .then((response) => response.json())
+  .then((json) => json);
 
-const renderWeatherDetails = (data) => {
-    const weatherDetailArea = document.getElementById("weather-info")
-    const weatherHtmlDetails = `
+const renderWeatherDetails = (data, container, toggle) => {
+  const fahrenheit = (data.main.temp * (9 / 5)) + 32;
```
[the remark this answers is not in the document]
Comment (tafodinho) [reply src/weather.js]: the idea here from the last review was to choose if i want my results to be in  Celsius or in Fahrenheit

[the remark this answers is not in the document]
Comment (rvvergara) [reply src/weather.js]: Kindly refer back to the project requirements/instructions :)

File: src/index.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (rvvergara) [reply src/index.js]: We are learning how to make our code optimal and use the more appropriate methods/events/scenarios. While it's good to try things but at the end of the day the reason why we are having code reviews is so we can improve our code.

[the remark this answers is not in the document]
Comment (tafodinho) [reply src/index.js]: Ok thanks am working on the appropriate changes

**Pull request merged**